parts/common/components.json

```diff
@@ -383,7 +383,21 @@
       "multiArchVersionsV2": [
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=azuremonitor/containerinsights/ciprod",
-          "latestVersion": "3.1.24"
+          "latestVersion": "3.1.25"
+        }
+      ],
+      "windowsVersions": [
+        {
+          "comment": "ciprod isn't supported on Win23H2 so is excluded as it makes the VHD too large",
```

Comment:
all ws2022 images should be able to run in 23H2. Does this mean that container insights will not support WindowsAnnualChannel?

Reply:
I was talking to the container insights team and they do not support windows annual.

The main problem with ciprod 3.1.25 is it's almost 1GB larger than 3.1.24 and wouldn't fit on the 23H2 image. So we agreed to remove ciprod from 23H2 so they can upgrade ciprod on the 2 windows versions they do support.

@wanlonghenry for visibility.